lzy/allocator/src/main/java/ai/lzy/allocator/alloc/dao/DynamicMountDao.java

```diff
@@ -0,0 +1,29 @@
+package ai.lzy.allocator.alloc.dao;
+
+import ai.lzy.allocator.model.DynamicMount;
+import ai.lzy.model.db.TransactionHandle;
+import org.jetbrains.annotations.Nullable;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public interface DynamicMountDao {
+    void create(DynamicMount dynamicMount, @Nullable TransactionHandle tx) throws SQLException;
+
+    @Nullable
+    DynamicMount get(String id, boolean forUpdate, @Nullable TransactionHandle tx) throws SQLException;
+
+    void delete(String id, @Nullable TransactionHandle tx) throws SQLException;
+
+    @Nullable
+    DynamicMount update(String id, DynamicMount.Update update, @Nullable TransactionHandle tx) throws SQLException;
+
+    long countForVolumeClaimName(String clusterId, String volumeClaimName, @Nullable TransactionHandle tx)
```

Comment:
count usages?

Reply:
For me, it sounds too ambiguous. _Count usages_ for what?

Reply:
Yeah, the main idea is to count how many mounts use specific volume claim. Maybe it should be named more clearly